insights/client/config.py

```diff
@@ -326,7 +326,7 @@ def _core_collect_default():
     'reregister': {
         'default': False,
         'opt': ['--force-reregister'],
-        'help': 'Forcefully reregister this machine to Red Hat. Use only as directed.',
+        'help': argparse.SUPPRESS,
```

Comment:
As we are in deprecation mode, I will not silence that option from `argparse` now. But I would suggest adding the deprecation message, announcing the ineffectiveness of that latter and its future removal.

Reply:
Agree, I have added a new message